gradle/libs.versions.toml

```
@@ -11,6 +12,7 @@ shadow = "com.github.johnrengelman.shadow:7.1.2"
 
 [libraries]
 kotlin-compiler = { module = "org.jetbrains.kotlin:kotlin-compiler-embeddable", version.ref = "kotlin" }
+dokka = "org.jetbrains.dokka:dokka-gradle-plugin:1.7.10"
```

Any reason not to do this?
```suggestion
dokka = { module = "org.jetbrains.dokka:dokka-gradle-plugin", version.ref = "kotlin" }
```

---

Cause Dokka was out later than Kotlin, we'd like to bump Kotlin first.

---

Ah, good to know, thanks!